app/routes/api+/asset.scan.ts

```diff
@@ -42,6 +42,7 @@ export async function action({ context, request }: ActionFunctionArgs) {
     const asset = await getAsset({
       id: assetId,
       organizationId,
+      userOrganizations,
```

Comment:
I don't think we should be doing this in an action. The way I imagine it is just for loaders.  Let me know if I am missing something.